packages/flutter/test/material/text_selection_test.dart

```diff
@@ -768,4 +768,24 @@ void main() {
     skip: isBrowser, // [intended] we don't supply the cut/copy/paste buttons on the web.
     variant: const TargetPlatformVariant(<TargetPlatform>{TargetPlatform.android}),
   );
-}
+
+  group('getHandleAnchor', () {
+    test('returns correct anchor for collapsed handle', () {
+      const Offset expected = Offset((22.0 - 2.0) / 2, -4.6);
+      final Offset actual = materialTextSelectionControls.getHandleAnchor(TextSelectionHandleType.collapsed, 0);
+      expect(actual, equals(expected));
+    });
+
+    test('returns correct anchor for left handle', () {
+      const Offset expected = Offset(22.0, 0);
+      final Offset actual = materialTextSelectionControls.getHandleAnchor(TextSelectionHandleType.left, 0);
+      expect(actual, equals(expected));
+    });
+
+    test('returns correct anchor for right handle', () {
+      const Offset expected = Offset.zero;
+      final Offset actual = materialTextSelectionControls.getHandleAnchor(TextSelectionHandleType.right, 0);
+      expect(actual, equals(expected));
+    });
+  });
+}
```

Comment:
I think these tests are not very useful since they just confirm the hardcoded values in the method. Could you add a test like I suggested in https://github.com/flutter/flutter/pull/169715#pullrequestreview-2898194726?